### DISCUSSION TITLE
生产者发送正常，MQ服务端消息丢失。RocketMQ-Console的Dashboard中Topic TOP 和 Broker TOP 都是0，但是TOP 5min trend是有数值的

### DISCUSSION THREAD

**Discussion by chenjunwenhao** [apache/rocketmq#8711] [Q&A]
现象：
      消费不到消息
排查过程：
    1.生产者发送返回正常。
    2.在RocketMQ-Console中查询Topic 的Message查询不到。
    3.RocketMQ-Console的Dashboard中Topic TOP 和 Broker TOP 都是0，但是TOP 5min trend是有数值的。
    4.commitlog中的00000000000中查看没有。
根据2，3，4怀疑是落盘出问题了

**Comment by luozongle01** [comment]
"commitlog中的00000000000中查看没有"   老哥你咋查看的commitlog啊，这个不都是二进制的吗。 那用./bin/mqadmin 查看topic的状态了吗，里面也显示没有消息吗

**Reply by chenjunwenhao** [reply]
commitlog中有二进制+消息体，其中也有消息体参杂在里。查看了没有。直观感受是服务端接到消息了，然后就直接丢弃了没落盘。

**Reply by luozongle01** [reply]
那没有落盘broker那边不应该有error日志吗   😳

**Reply by luozongle01** [reply]
> commitlog中有二进制+消息体，其中也有消息体参杂在里。查看了没有。观察感受器是服务端接收到消息了，然后就直接丢弃了没落盘。
broker那边一点错误日志都没吗 😅

**Reply by chenjunwenhao** [reply]
broker 有个主 从同步的 ACL认证失败日志，其他就没有了。
